config/default_records/submission_templates/009_scrna_core_cell_extraction_submission_templates.wip.yml

```diff
@@ -6,7 +6,6 @@ Limber - scRNA Core Cell Extraction:
   submission_class_name: "LinearSubmission"
   related_records:
     request_type_keys: ["limber_scrna_core_cell_extraction"]
-    project_name: "UAT Project"
-    study_name: "UAT Study"
+    project_name: "S0842"
```

Comment:
N.B. had to delete the submission template, and call `WIP=009_scrna_core_cell_extraction_submission_templates bundle exec rake record_loader:all` to get this to update.

Reply:
TODO: Integration Suite test update.

Reply:
Update: Looks like Integration Suite update not necessary.

Reply:
That looks like a cost code rather than a project name, is it correct to use that?

Reply:
That's how Rich C set it up in SS - how he'd like them all to be.